src/ert/gui/simulation/view/realization.py

```diff
@@ -153,3 +162,24 @@ def _paint_inner_grid(painter: QPainter, rect: QRect, colors) -> None:
 
     def sizeHint(self, option, index) -> QSize:
         return self._size
+
+    def eventFilter(self, watched, event: QEvent):
+        if event.type() == QEvent.Type.ToolTip:
+            mouse_pos = event.pos() + QPoint(
+                -2 * self.job_rect_margin, -2 * self.job_rect_margin
```

Comment:
Propose to move this `QPoint` as an instance variable, -- we don't need to recreate this object every time this function is called.

Reply:
Agreed!